nerder/gh-explorer
# #8: main layout structure (closes #8)

Issue #8 
## Test Plan
- Edit HTML and add to the second `<div>` of the list a margin-top of 1000px, you will see a vertical scroll bar.
- Edit HTML and add to the second `<div>` of the list a margin-left of 1100px (or anything >= 1001px) you will see the element disappear but no horizontal scroll bar.
- Resize the window to be less than 1000px of width you will see the results `<div>` without any borders
